packages/salesforce-adapter/src/config_change.ts

```diff
@@ -27,6 +27,9 @@ const MESSAGE_INTRO = 'Salto failed to fetch some items from salesforce. '
 const MESSAGE_REASONS_INTRO = 'Due to the following issues: '
 const MESSAGE_SUMMARY = 'In order to complete the fetch operation, '
 + 'Salto needs to stop managing these items by applying the following configuration change:'
+export const DEPRECATED_OPTIONS_MESSAGE = 'The configuration options "metadataTypesSkippedList", "instancesRegexSkippedList" and "dataManagement" are deprecated.'
++ ' The following changes will update the deprected options to the "fetch" configuration option.'
```

Comment:
```suggestion
+ ' The following changes will update the deprecated options to the "fetch" configuration option.'
```

might want to install a [spell check plugin](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to catch these kinds of typos more easily